src/NumericalAlgorithms/Interpolation/InterpolationTargetKerrHorizon.cpp

```diff
@@ -14,11 +14,13 @@ namespace OptionHolders {
 KerrHorizon::KerrHorizon(size_t l_max_in, std::array<double, 3> center_in,
                          double mass_in,
                          std::array<double, 3> dimensionless_spin_in,
+                         const bool theta_varies_fastest_in,
```

Comment:
why is this needed?

Reply:
It needs to be specifiable by options, because the ordering of points is different for the apparent horizon interface and for CCE -- or have I misunderstood the question?

Reply:
Nope, that was the confusion. Maybe document that just as a "for example" usage case so it's recorded somewhere?